tests/README.md

```diff
@@ -38,7 +38,7 @@ correctly, or to extract the correct name to run a one shot test.
 To manually run one shot tests, first [connect to a cluster](https://console.cloud.google.com/kubernetes/list) and then run the following:
 
 ```bash
-export TEST_NAME=tf.nightly-resnet-imagenet-func-v2-8
+export TEST_NAME=tf.nightly-vit-beans-func-v4-8-1vm
 jsonnet tests/oneshot.jsonnet -J . -S --tla-str test=$TEST_NAME | kubectl create -f -
 ```
 
```

Comment:
I think we should also adjust the job id to match with the test_name used in the one shot tutorial

Reply:
Done